packages/admin-ui-docs/docs/guidelines/components/bulk-actions.mdx

```diff
@@ -0,0 +1,242 @@
+---
+title: Bulk Actions
+path: /bulk-actions/
+hide_table_of_contents: true
```

Comment:
I think you should remove this line ;) because you are not using the page + tabs template ;)